README.md

```diff
@@ -28,19 +28,18 @@ Current version of the [pipeline](https://github.com/phenoscape/phenoscape-owl-t
 ***
 
 ## Deployment
+1. Download docker image `phenoscape/pipeline-tools` from [here](https://hub.docker.com/r/phenoscape/pipeline-tools).
```

Comment:
I don't believe this is necessary. It just happens automatically.

Reply:
Oh, I didn't know that. That's convenient.